core/management.c

```diff
@@ -17,7 +17,8 @@
  *    Bosch Software Innovations GmbH - Please refer to git log
  *    Pascal Rieux - Please refer to git log
  *    Tuve Nordius, Husqvarna Group - Please refer to git log
- *    
+ *    Frederic Dur - Please refer to git log
```

Comment:
Probably not a big deal, but I guess the author here should rater be the author of the code. 

My concern is about potential IP issue. (even if probability is very low here)
In particular because  "There is no valid ECA on file for fdur@sierrawireless.com." (checked with ECA Validation Tool of eclipse)

(Out of topic but to be honest I'm not fan of this kind of log in the header especially when we have tool like git history. Just to let you know this is not mandatory for eclipse project see [Tracking contributors in Git vs file header comment](https://bugs.eclipse.org/bugs/show_bug.cgi?id=435933#c18) but project is free to use it if wanted)

Reply:
In this case I reasoned that the change was trivial once the problem was pointed out and it was nicer to give some credit than not to, however I see the general problem. And I agree about the header comments.
Correct me if I am wrong but pointing out a problem doesn't give anyone any ownership of the solution. If the solution is obvious and trivial there is no IP either - anyone can come to the same result.
I suggest we close this PR (might as well close #440 also). I can submit the obvious fix in a new PR.
(making an effort to reduce the number of open TRs)

Reply:
As I said my 'little' concern is just about this line in the header.

**I'm pretty sure there is no issue** but if there is that could be impacting, like e.g. we need to retrieve Frederic Dur to get a given agreement about license changing or something like this. 

I just say that maybe this is safer to avoid this and so just thanks to the contributor for reporting the issue.

But again I'm not sure of all of this. And I just give you some concern. Feel free to let it like this or ask to other member of the team.

If you finally want to adapt the PR, just push force this one rather than create a new one.

Reply:
Force pushed a brand new commit - 100% handcrafted by me (21 characters of hard work 😂). Please merge!

Reply:
:grin: